### DISCUSSION TITLE
Anyways to acquire microphone permission on macOS using Avalonia's API?

### DISCUSSION THREAD

**Discussion by laolarou726** [AvaloniaUI/Avalonia#10799] [Q&A]
Are there any ways to request microphone permission for an Avalonia application on macOS? I didn't find related documentation. I tried to import Xamarin.Mac into my project but it doesn't work.

**Comment by ahopper** [comment]
You need to include 'Privacy - Microphone Usage Description' in your info.plist and sign with com.apple.security.device.audio-input entitlement
